### PR TITLE
Gotts - Use Prod API

### PR DESCRIPTION
**Pull request by sebko**

### CODE REVIEW

File: gotts-menu/pages/menu-boards/eats-and-drinks.tsx [hunk 1]
```diff
@@ -65,7 +64,7 @@ const EatsAndDrinks: NextPage<EatsAndDrinksProps> = ({ data }) => {
 
   return (
     <MainLayout
-      footerText={String(footerText)}
+      footerText={footerText as string | undefined}
```
Comment (sebko) [inline gotts-menu/pages/menu-boards/eats-and-drinks.tsx:67]: When omitting the query string `undefined` was being converted to `"undefined"` which caused this to happen
image

File: gotts-menu/pages/menu-boards/eats-and-drinks.tsx [hunk 2]
```diff
@@ -103,6 +102,13 @@ const EatsAndDrinks: NextPage<EatsAndDrinksProps> = ({ data }) => {
               />
             ))}
           </Group>
+
+          <Box as="section" mb={4}>
+            <Paragraph sx={{ fontSize: 0 }}>Wine corkage $5.00</Paragraph>
+            <Paragraph sx={{ fontSize: 0 }}>
+              Vintages are available upon request.
+            </Paragraph>
+          </Box>
```
Comment (sebko) [inline gotts-menu/pages/menu-boards/eats-and-drinks.tsx:111]: image

File: gotts-menu/pages/menu-boards/eats-and-drinks.tsx (hunk 1 above)
Comment (jschr) [reply gotts-menu/pages/menu-boards/eats-and-drinks.tsx:69]: `footerText ?? ''` Might work better.

In general we should avoid manual casting with `as`. Alternatively you could change the type in `MainLayout` to match the type of `footerText` returned by `router.query`.

Comment (sebko) [reply gotts-menu/pages/menu-boards/eats-and-drinks.tsx:69]: This is an interesting one. The `footerText` prop for `MainLayout` accepts either a `string` or `undefined`. However,  the type for `router.query.footerText` is `string | string[] | undefined`.

I tried `footerText ?? ''` but received the following error message `Type 'string[]' is not assignable to type 'string'.`.

We could change the type in `MainLayout` to match the type of `footerText` returned by `router.query` but that would include the `string[]` type, which technically isn't supported by `MainLayout`. Probably fine to do that though.

Comment (jschr) [reply gotts-menu/pages/menu-boards/eats-and-drinks.tsx:69]: Ahh.. right yeah that's a bit annoying.

Would need to do something like `Array.isArray(footerText) ? footerText[0] : footerText`. A bit tedious though. In the past I've created a helper function for it but casting isn't terrible in this case either. Up to you!

**Pull request merged**